.prettierrc.json

```diff
@@ -1,4 +1,5 @@
 {
     "printWidth": 100,
-    "tabWidth": 4
+    "tabWidth": 4,
+    "trailingComma": "none"
```

Comment:
same here, did the default change?

Reply:
Yes, Prettier v2 has now trailing commas set to "es5" by default, before it was "none"
https://prettier.io/docs/en/options.html#trailing-commas